gpflow/utilities/utilities.py

```diff
@@ -364,15 +364,16 @@ def setattr_by_path(target: object, attr_path: str, value: Any):
 M = TypeVar("M", bound=tf.Module)
 
 
-def deepcopy(input_module: M) -> M:
+def deepcopy(input_module: M, memo: Optional[Dict[int, Any]] = None) -> M:
     """
     Returns a deepcopy of the input tf.Module. To do that first resets the caches stored inside each
     tfp.bijectors.Bijector to allow the deepcopy of the tf.Module.
 
     :param input_module: tf.Module including keras.Model, keras.layers.Layer and gpflow.Module.
+    :param memo: Check details for func:`copy.deepcopy` `memo` argument.
```

Comment:
```suggestion
    :param memo: passed through to func:`copy.deepcopy` (see https://docs.python.org/3/library/copy.html).
```
The docstring of the `copy` module and `copy.deepcopy` itself are actually rather unhelpful...